ecs-cli/modules/commands/compose/service/compose_service_command.go

```diff
@@ -189,3 +193,14 @@ func loadBalancerFlags() []cli.Flag {
 		},
 	}
 }
+
+// ComposeServiceTimeoutFlag allows user to specify a custom timeout
+func ComposeServiceTimeoutFlag() cli.Flag {
+	return cli.Float64Flag{
+		Name:  command.ComposeServiceTimeOutFlag,
+		Value: service.DefaultUpdateServiceTimeout,
+		Usage: fmt.Sprintf(
+			"Specifies the timeout value in minutes (decimals supported) to wait for the running task count to change. If the running task count has not changed for the specified period of time, then the CLI times out and returns an error. The default timeout value is 5 minutes. Setting the timeout to 0 will cause the command to return without checking for success.",
```

Comment:
is "The default timeout value is 5 minutes" needed? isn't there a (default: 5) at the end of the message already?